dags/hivemind_etl_helpers/tests/integration/test_gdrive_loader.py

```diff
@@ -218,10 +217,11 @@ def test__load_by_file_id(self, mock_load_data):
         ]
         mock_load_data.return_value = mock_data
         loader = GoogleDriveLoader(self.mock_client_config)
-        result = loader._load_from_files(file_id=file_ids)
+        result = loader._load_from_files(file_ids=file_ids)
 
         self.assertEqual(len(result), 2)
         self.assertEqual(result, mock_data)
+        assert result[0].id_ == mock_data[0].id_
```

Comment:
Please consider checking for result length too. You can use the self.assertEqual the same as other parts of your code